packages/docs/getting-started/running-a-validator-in-rc1.md

```diff
@@ -153,40 +153,131 @@ The setup of RC1 is similar to the new Baklava network and the deployment timeli
 A [timeline](https://forum.celo.org/t/release-candidate-1-rc1-timeline-and-details/428) of the Release Candidate 1 is available to provide further context.
 {% endhint %}
 
-## Setup for Genesis Validators (Before 4/22)
+## Instructions Before Producing Blocks
 
-**If you provided your Validator signer address and BLS public key for genesis, the community is relying on your Validator to get the network started!**
-
-This section outlines the steps needed to configure your Proxy and Validator nodes before block production begins.
+This section outlines the steps needed to configure your Proxy and Validator nodes before you can start producing blocks.
 
 ### Environment Variables
 
-First we are going to set up the main environment variables related to the RC1 network. Run these on both your **Validator** and **Proxy** machines:
+First we are going to set up the main environment variables related to the RC1 network. Run these on all machines:
 
 ```bash
 export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node:rc1
 export NETWORK_ID=42220
-export CELO_VALIDATOR_SIGNER_ADDRESS=<YOUR-VALIDATOR-SIGNER-ADDRESS>
 ```
 
-Please use the Validator signer address that you submitted through your Gist file. It is included in the [genesis Validator set](https://github.com/celo-org/celo-monorepo/blob/asaj/rc1/packages/celotool/genesis_rc1.json). If you plan to run additional validators once the network is up and running you will need to create additional validator signer addresses.
-
 ### Pull the Celo Docker image
 
-In all the commands we are going to see the `CELO_IMAGE` variable to refer to the Docker image to use. Now we can get the Docker image on your Validator and Proxy machines:
+In all the commands we are going to see the `CELO_IMAGE` variable to refer to the Docker image to use. Now we can get the Docker image on all machines:
 
 ```bash
 docker pull $CELO_IMAGE
 ```
 
 The `us.gcr.io/celo-testnet/celo-node:rc1` image contains the [genesis block](https://github.com/celo-org/celo-monorepo/blob/asaj/rc1/packages/celotool/genesis_rc1.json) in addition to the Celo Blockchain binary.
 
+### Create the Validator and Validator Group accounts
+
+First, you'll need to generate account keys for your Validator and Validator Group.
+
+{% hint style="danger" %}
+These keys will become the beneficiary addresses of your `ReleaseGold` contracts, and thus should be handled with care.
+
+Store and back these keys up in a secure manner, as there will be no way to recover if them if lost or stolen.
+{% endhint %}
+
+```bash
+# On your local machine
+mkdir celo-accounts-node
+cd celo-accounts-node
+docker run -v $PWD:/root/.celo --rm -it $CELO_IMAGE account new
```

Comment:
should creating the accounts via the node be encouraged? Feels like celocli + useLedger would be ideal

Reply:
added notes below. @asaj do you think we should show the `account new` command at all?

Reply:
Happy either way, though agree with @pranaymohan's suggestion, a ledger would be best

Reply:
added a warning